util/diff_runs.py

```diff
@@ -0,0 +1,244 @@
+#!/usr/bin/env python3
```

Comment:
Does this script need Python 3 features?

Reply:
No, it needs python >= 2.7.11. Ubuntu dist is 2.7.6, making it a difficult process to jump through hoops (though, once we have docker running, this becomes redundant)